### DISCUSSION TITLE
Make it clear Upptime is not affiliated to GitHub

### DISCUSSION THREAD

**Discussion by K3UL** [upptime/upptime#143] [Ideas] [answered]
I suggest making it clear that your tool is powered by GitHub **features**, and not by GitHub themselves. I would change the short description of the repo, and add a disclaimer of non-affiliation to the readme file.
IMO it is confusing at first sight.

**Comment by AnandChowdhary** [comment, marked as answer]
I've updated the repo description and README. I'll also get around to updating the landing page soon. Thanks for pointing this out!
